persistent/Database/Persist/Sql/Orphan/PersistQuery.hs

```diff
@@ -249,6 +249,12 @@ filterClauseHelper includeTable includeWhere conn orNull filters =
                            let xxs = transpose (map fromPersistList allVals)
                                sqls=map (\(a,xs) -> connEscapeName conn (fieldDB a) <> showSqlFilter pfilter <> "(" <> T.intercalate "," (replicate (length xs) " ?") <> ") ") (zip (compositeFields pdef) xxs)
                            in (wrapSql (T.intercalate " or " (map wrapSql sqls)), concat xxs)
+                        ([PersistList xs], Like, _) ->
+                           let sqlcl=T.intercalate " and " (map (\a -> connEscapeName conn (fieldDB a) <> showSqlFilter pfilter <> "?" <> " ESCAPE ? ") (compositeFields pdef))
```

Comment:
Formatting nitpick: Can we put spaces around the `=` and try to keep the line length down
```suggestion
                           let sqlcl = T.intercalate " and " 
                                   . map (\a -> mconcat
                                       [ connEscapeName conn (fieldDB a)
                                       , showSqlFilter pfilter
                                       , "?", " ESCAPE ? ") 
                                       ]
                                   $ compositeFields pdef
```